keywords/SyncGateway.py

```diff
@@ -224,7 +224,7 @@ def start_sync_gateways(self, cluster_config, url=None, config=None):
 
         if is_xattrs_enabled(cluster_config):
             playbook_vars["autoimport"] = '"import_docs": "continuous",'
-            playbook_vars["xattrs"] = '"enable_extended_attributes": true'
+            playbook_vars["xattrs"] = '"enable_shared_bucket_access": true,'
```

Comment:
comma is needed. In the json, {{ xattrs }} gets replaced with "enable_shared_bucket_access": true, . Without comma, it would be an invalid json.